platform/darwin/src/NSValue+MGLStyleEnumAttributeAdditions.h

```diff
@@ -0,0 +1,150 @@
+// This file is generated.
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import <Foundation/Foundation.h>
+
+#import "MGLFillStyleLayer.h"
+#import "MGLLineStyleLayer.h"
+#import "MGLSymbolStyleLayer.h"
+#import "MGLCircleStyleLayer.h"
+#import "MGLRasterStyleLayer.h"
+#import "MGLBackgroundStyleLayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSValue (MGLStyleEnumAttributeAdditions)
```

Comment:
realm/jazzy#539 means that either all of NSValue (MGLAdditions) will wind up in this category, or all of this category will wind up in NSValue (MGLAdditions). Hopefully the latter.